src/web/seo/favicons.astro

```diff
@@ -0,0 +1,4 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png" />
+<link rel="icon" type="image/png" sizes="32x32" href="/favicons/favicon-32x32.png" />
+<link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png" />
+<link rel="manifest" href="/site.webmanifest" />
```

Comment:
Nie brakuje tutaj czasem odwołania do `favicon.ico`?

Reply:
Bazując na tym co jest napisane w tym [konwerterze](https://favicon.io/favicon-converter/) w sekcji `Installation` to nie. Favicon wyświetla się poprawnie i zgaduje, że przeglądarka bierze sobie odpowiedni size, w zależności od tego gdzie ten favicon jest wyświetlany.